.circleci/config.yml

```diff
@@ -152,27 +152,23 @@ commands:
       k8s_image_name:
         type: string
         default: "deriv-app"
+      k8s_namespace:
+        type: string
+        default: "deriv-app-staging"
     steps:
       - k8s/install-kubectl
       - run:
-          name: Deploying to k8s cluster for service deriv-app
+          name: Deploying to k8s cluster for service << parameters.k8s_namespace >>
           command: |
             TAG=${CIRCLE_TAG:-$CIRCLE_SHA1}
-            for SERVER_ID in {1..5}
-            do
-              KUBE_SERVER_REF="KUBE_SERVER_$SERVER_ID"
-              SERVICEACCOUNT_TOKEN_REF="SERVICEACCOUNT_TOKEN_$SERVER_ID"
-              CA_CRT_REF="CA_CRT_$SERVER_ID"
-              if [ ! -z "${!KUBE_SERVER_REF}" ]
-              then
-                echo "Deploying to cluster $SERVER_ID"
-                CA_CRT="${!CA_CRT_REF}"
-                KUBE_SERVER="${!KUBE_SERVER_REF}"
-                SERVICEACCOUNT_TOKEN="${!SERVICEACCOUNT_TOKEN_REF}"
-                echo $CA_CRT | base64 --decode > ca.crt
-                kubectl --server=${KUBE_SERVER} --certificate-authority=ca.crt --token=$SERVICEACCOUNT_TOKEN set image deployment/<< parameters.k8s_svc_name >> << parameters.k8s_svc_name >>=${DOCKHUB_ORGANISATION}/<< parameters.k8s_image_name >>:${TAG}
-              fi
-            done
+            echo $CA_CRT | base64 --decode > ca.crt
+            kubectl --server=${KUBE_SERVER} --certificate-authority=ca.crt --token=$SERVICEACCOUNT_TOKEN set image deployment/<< parameters.k8s_svc_name >> << parameters.k8s_svc_name >>=${DOCKHUB_ORGANISATION}/<< parameters.k8s_image_name >>:${TAG}
```

Comment:
I don't think this should be here still?

I was expecting it to look more like [this](https://github.com/binary-com/deriv-com/blob/master/.circleci/config.yml#L133-L137)

Reply:
After switching traffic to blue/green these two line will be deleted.

Reply:
🤦  I just realised you're doing *exactly* what I said to do earlier.
Dual deployments initially to the old namespace.

Okay, so I *think* just adding --namespace=default to this would have saved me from not realising that.

But I'll approve anyway.